recipe/meta.yaml

```diff
@@ -30,6 +34,12 @@ test:
     - dash_mantine_components.Image
     - dash_mantine_components.Center
     - dash_mantine_components.Tooltip
+  commands:
+    - conda install -c conda-forge dash-html-components dash-core-components dash-table -y
```

Comment:
Can you add a comment explaining what this is supposed to test/prevent?

Reply:
Done - Referenced discussion from upstream feed stock.